templates/zerver/for/open-source.md

```diff
@@ -1,3 +1,12 @@
+[Zulip Standard](/plans) is free for open-source communities! To join the
+hundreds of open-source projects we sponsor, you can request sponsorship
+from your organization's
+[billing page](/accounts/go/?next=/upgrade%23sponsorship), or contact us at
+sales@zulip.com to check whether you organization qualifies.
```

Comment:
Minor typo!
'you' -> 'your'. 

Context: 
<i>to check whether your organization qualifies.</i>